### PR TITLE
Log codedeploy results and show those messages

### PR DESCRIPTION
**Pull request by foostan**
Verification of log output is complete, so enable it.

### CODE REVIEW

File: kode [hunk 1]
```diff
@@ -485,8 +485,7 @@ tasks:
         -e "KUBE_DNS_SERVICE_HOST=${KUBE_DNS_SERVICE_HOST}" \
         -e "KUBE_DNS_SERVICE_PORT=${KUBE_DNS_SERVICE_PORT}" \
         --rm -w "${wd}" \
-        "${image}" bash -c "${cmd}"
-        # | cloudwatch-logger -t "${log_group}" ${DEPLOYMENT_ID/stdout}
+        "${image}" bash -c "${cmd}" | cloudwatch-logger -t "${log_group}" ${DEPLOYMENT_ID/stdout}
```
Comment (mumoshu) [inline kode:488]: I think `${DEPLOYMENT_ID/stdout}` should be `${DEPLOYMENT_ID}/stdout` and `${deploy_id}` in line 615 to be `${deploy_id}/stdout`, but LGTM overall!

Comment (mumoshu) [reply kode:488]: Ah, so bash evaluates `${DEPLOYMENT_ID/stdout}` to just `${DEPLOYMENT_ID}`, so this should work anyway, for stdout only. LGTM.

**Review by mumoshu — APPROVED** [review]

**Comment by mumoshu** [comment]
@foostan Thanks for your contribution as always ☺️

File: kode (hunk 1 above)
Comment (foostan) [reply kode:488]: Thanks for checking about bash evaluations 👍

**Pull request merged**